pubspec.yaml

```diff
@@ -0,0 +1,25 @@
+name: workmanager_tizen
+description: A new Flutter plugin project.
```

Comment:
Please write this as well as `description`.

Reply:
I wrote description "Tizen plugin of WorkManager." is this okay?